public/realms/devnet.json

```diff
@@ -128,7 +128,7 @@
   },
   {
     "symbol": "LEO",
-    "displayName": "LeoDao New York City",
+    "displayName": "LeoDaoNewYorkCity _ DEMO",
     "programId": "GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw",
```

Comment:
the DAO is created using `GTesTBiEWE32WHXXE2S4XbZvA5CrEc4xs6ZgRe895dP` instance

Reply:
Okay thanks,. Was not aware you could not change that through the JSON. Will update/create a new request

Reply:
corrected the programId and pushed it again.